libs/react-helpers/src/lib/grid/price-flash-cell.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import * as React from 'react';
```

Comment:
chore (non-blocking): React 17 we shouln't need to import react unless its used, same goes for other files in the PR. (PS I realize we are unnecessarily importing it in other files 🕊️ )
```suggestion

```